apps/code-of-conduct/app/user-table/user-table.component.html

```diff
@@ -0,0 +1,43 @@
+<mat-table [dataSource]="blockService.blockedUsers">
+  <ng-container matColumnDef="username">
+    <mat-header-cell *matHeaderCellDef> Username </mat-header-cell>
+    <mat-cell class="username-cell" *matCellDef="let user">
+      <img src="https://github.com/{{user.get('username')}}.png">
+      <span>{{user.get('username')}}</span>
+    </mat-cell>
+  </ng-container>
+
+  <ng-container matColumnDef="blockUntil">
+    <mat-header-cell *matHeaderCellDef> Blocked Until </mat-header-cell>
+    <mat-cell *matCellDef="let user"> {{user.get('blockUntil') === false ? 'Blocked Indefinitely' : user.get('blockUntil').seconds * 1000 | date}} </mat-cell>
```

Comment:
I see you are doing the *1000 unix conversion here again. Why can't these table rows not also use the `BlockedUser` converter that already does this?